app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/ActionHeader.js

```diff
@@ -135,65 +142,35 @@ const ActionHeader = ({
-                  setIsFiltersCollapsed((prev) => !prev);
-                  trackTrafficTableFilterClicked();
-                }}
-                className={isFiltersCollapsed ? "traffic-table-filter-btn-inactive" : "traffic-table-filter-btn-active"}
-              />
-            </Badge>
+            <Button icon={<FilterOutlined />} onClick={handleFilterClick} disabled={isDisableActionButton}>
+              <span className="traffic-table-filter-btn-content">
+                <span>Filter</span>
+                <Badge className="traffic-table-applied-filter-count" count={activeFiltersCount} size="small" />
+              </span>
+            </Button>
           </Col>
           {isStaticPreview ? null : (
             <>
               <Col>
-                <Tooltip placement="top" title="Clear Logs">
-                  <Button
-                    type="primary"
-                    disabled={!logsCount}
-                    shape="circle"
-                    icon={<ClearOutlined />}
-                    onClick={clearLogs}
-                  />
-                </Tooltip>
+                <Button icon={<ClearOutlined />} onClick={clearLogs} disabled={!logsCount || isDisableActionButton}>
+                  Clear log
+                </Button>
               </Col>
-              {isFeatureCompatible(FEATURES.NETWORK_SESSIONS) && isImportNetworkSessions ? (
-                <>
-                  <Col>
-                    <Tooltip placement="top" title="Save Network Session">
-                      <Button
-                        type="primary"
-                        disabled={!logsCount}
-                        icon={<SaveOutlined />}
-                        onClick={() => {
-                          trackNetworkSessionSaveClicked();
-                          openSaveModal();
-                        }}
-                      />
-                    </Tooltip>
-                  </Col>
-                  <Col>
-                    <Tooltip placement="top" title="Export Network Session">
-                      <Button
-                        type="primary"
-                        disabled={!logsCount}
-                        icon={<DownloadOutlined />}
-                        onClick={() => {
-                          downloadHar(logsToSaveAsHar || {}, "");
-                          trackDownloadNetworkSessionClicked(ActionSource.TrafficTable);
-                        }}
-                      />
-                    </Tooltip>
-                  </Col>
-                </>
-              ) : null}
+
+              <Col>
+                <PauseAndPlayButton
+                  defaultIsPaused={false}
+                  isDisableActionButton={isDisableActionButton} // only disable resume button when app not connected
+                  onChange={(isPaused) => {
+                    setIsInterceptingTraffic(!isPaused);
+                  }}
+                  disabled={!logsCount || isDisableActionButton}
```

Comment:
Here the two disable props become very confusing. either just pass the `isAppConnected` state as argument and let this component decided on its own

or rename and create two new props `disablePause` and `disablePlay` (this would be better)